pkg/server/nfs/nfs_server.go

```diff
@@ -29,8 +28,18 @@ NFS_Core_Param
     Protocols = 4;
 }
 
-# uncomment to enable debug logging
-# LOG { COMPONENTS { NFS_V4 = FULL_DEBUG; } }
+LOG {
+	Default_Log_Level = INFO;
+
+# 	uncomment to enable debug logging
+#	COMPONENTS { NFS_V4 = FULL_DEBUG; }
```

Comment:
Will this cause some debugging logs missing? Right now, there will be only info log will be logged except any error raised.

Reply:
Following this pattern will be more clear.
```
LOG {
    ## Default log level for all components
    Default_Log_Level = INFO;

    ## Configure per-component log levels
    Components {
            NFS4 = ...;
             ...
    }

    ## Where to log
    Facility {
            name = FILE;
            destination = "....";
            enable = active;
    }
}
```

Reply:
The configurable log level will be addressed in https://github.com/longhorn/longhorn/issues/4420.

Reply:
Since it won't over write the configuration file if already exists.
https://github.com/longhorn/longhorn-share-manager/blob/438c1ae100013b6b68c89b3a277d2cb9b790fc3c/pkg/server/nfs/nfs_server.go#L91-L95

I wonder if we can provide the configuration file as sample whose Default_Log_Level is FULL_DEBUG  and set the destination path as `/tmp/ganesha.log`

Reply:
I try to add one function to replace the log level based on parameter that assigned in main function. Please help to [the comment in #4420](https://github.com/longhorn/longhorn/issues/4420#issuecomment-1217818701)

Reply:
The idea is good, but I would suggest keeping it as it for now, because our goal is just to redirect the log for now.